src/SFA.DAS/Authorization/FeatureType.cs

```diff
@@ -1,19 +1,19 @@
 namespace SFA.DAS.Authorization
 {
-public enum FeatureType
-{
-NotSpecified,
-Activities,
-Projections,
-ProviderRelationships,
-Recruitments,
-TransferConnectionRequests,
-Transfers,
-EnableNewRegistrationJourney,
-HomePageFeature,
+    public enum FeatureType
+    {
+        NotSpecified,
+        Activities,
+        Projections,
+        ProviderRelationships,
+        Recruitments,
+        TransferConnectionRequests,
+        Transfers,
+        EnableNewRegistrationJourney,
```

Comment:
Are we keeping the feature enum value in if it has no purpose going forward?

Reply:
Doesn't run if it's removed.